src/ProgramBlockEditor.test.js

```diff
@@ -457,14 +646,14 @@ test('The editor scrolls when a step is added to the end of the program', () =>
 
     window.HTMLElement.prototype.scrollIntoView = mockScrollIntoView;
 
-    // Given a program of 5 forwards and 'forward' as the selected command
+    // Given a program of 5 forwards and 'left' as the selected command
     const { wrapper, audioManagerMock, mockChangeProgramHandler } = createMountProgramBlockEditor({
         program: ['forward', 'forward', 'forward', 'forward', 'forward'],
-        selectedAction: 'forward'
+        selectedAction: 'left'
     });
 
     // When 'forward' is added to the end of the program
-    const addNode = getAddNodeButtonAtPosition(wrapper, 5);
+    const addNode = getAddNodeButtonAtPosition(wrapper, 0);
```

Comment:
Thanks very much for fixing this test Tony. Could you also add an explanatory note as to why we are using index 0 here, as you did for your new test `"We should be able to add a step at the end of the program"`:

    // (The index is zero because the add nodes aren't expanded).

Reply:
Done, throughout.